src/styles/blocks/header.scss

```diff
@@ -0,0 +1,12 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  width: calc(100% - (50px * 2));
```

Comment:
The width calculation `calc(100% - (50px * 2))` is correct but could be simplified to `calc(100% - 100px)` for clarity. This change will not affect functionality but can make the code easier to read.